tests/monday_types/people_type_tests.py

```diff
@@ -0,0 +1,182 @@
+import json
+from schematics.exceptions import ConversionError, DataError
+from nose.tools import eq_,raises
+
+from moncli import entities as en
+from moncli.entities.column_value.objects import PersonOrTeam
+from moncli.enums import ColumnType, PeopleKind
+from moncli.models import MondayModel
+from moncli.types import PeopleType
+
+
+def test_should_succeed_when_to_native_returns_a_list_when_passing_in_peoplevalue_value_with_api_data_and_max_allowed_is_not_1():
+
+    # Arrange
+    id = "people_1"
+    title = 'Assignee'
+    column_type = ColumnType.people
+    value = {'personsAndTeams': [
+        {'id': 13435, 'kind': 'person'}, {'id': 11234, 'kind': 'person'}]}
+    value = json.dumps(value)
+    settings_str = json.dumps({'max_people_allowed': 0})
+    people_value = en.cv.create_column_value(column_type,id=id,title=title,value=value,settings_str=settings_str)
+
+    # Act
+    people_type = PeopleType(title='Assignee')
+    value = people_type.to_native(people_value)
+
+    # Assert
+    eq_(value[0].id, 13435)
+    eq_(value[1].kind, PeopleKind.person)
+
+def test_should_suceed_when_to_native_returns_a_personorteam_when_passing_in_a_peoplevalue_with_api_data_and_max_allowed_is_1_to_people_type():
+    
+    # Arrange
+    people_type = PeopleType(title='Assignee',max_allowed=1)
+
+    # Act
+    value = people_type.to_native(en.cv.PersonOrTeam(id=12345,kind=PeopleKind.person))[0]
```

Comment:
When max_allowed is 1, to_native should not return a list.  Therefore you do not need the index subscripting for [0] at the end.  Please remove that.